lib/model/query/entities.js

```diff
@@ -110,16 +111,26 @@ const createMany = (dataset, entities, sourceId, userAgentIn) => async ({ all, c
     userAgent,
     'true',
     'true',
-    sql`clock_timestamp()`,
     '1'
-  ], sql`,`)} )`), sql`,`);
+  ]);
+  const defColumnTypes = [
+    'int4',
+    'text',
+    'json',
```

Comment:
Maybe it'd be better for this to be `jsonb`, since that's what the column ultimately is.